include/ua_types.h

```diff
@@ -28,6 +28,12 @@ extern "C" {
 #endif
 #include "ua_config.h"
 #include "ua_statuscodes.h"
+  
+/*************************/
+/* External Dependencies */
+/*************************/
+#include "pcg_basic.h"
```

Comment:
Now we don't need this include in the public interface. It can be moved to ua_types.c.

Otherwise, ready to merge!